daemon/lib/shell.go

```
@@ -77,8 +81,23 @@ func ExecCommandOutputWithContext(ctx context.Context, command string, args ...s
+	if err != nil {
+		return string(out), fmt.Errorf("command failed: %w", err)
+	}
+	return string(out), nil
+}
+
 // CommandExists checks if a command exists in PATH
 func CommandExists(command string) bool {
 	_, err := exec.LookPath(command)
 	return err == nil
-}
+}
```

golangci-lint (2.11.4)

[error] File is not properly formatted

(gofmt)